modules/bmo-about-the-firmwareschema-resource.adoc

```diff
@@ -0,0 +1,46 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: REFERENCE
+[id="about-the-firmwareschema-resource_{context}"]
+= About the FirmwareSchema resource
+
+BIOS settings vary among hardware vendors and host models. A `FirmwareSchema` resource contains the limits for each BIOS setting on each host. The data comes directly from the BMC through Ironic. The `FirmwareSchema` prevents you from sending invalid values to a host when specifying values in the `spec` field of the `HostFirmwareSettings` resource. The `FirmwareSchema` resource has a unique identifier derived from its settings and limits. Identical host models may use the same `FirmwareSchema` identifier. So it is likely that the `HostFirmwareSettings` resource may contain multiple instances of the `FirmwareSchema` with the same identifier.
```

Comment:
*contains the limits* should be *contains the type and limits*

This statement is confusing - "So it is likely that the `HostFirmwareSettings` resource may contain multiple instances of the `FirmwareSchema` with the same identifier."   It should something like "It is likely that the same FirmwareSchema is used by multiple instances of HostFirmwareSettings